### DISCUSSION TITLE
storage is driving me crazy

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#5743] [Questions]
I deleted all images from storage.

Why i still get image with this?
![image](https://user-images.githubusercontent.com/56837461/156256241-9dafb649-49b7-4afd-9430-aa9628717414.png)

path is from database.

**Comment by GaryAustin1** [comment]
Most likely they are still cached until the cache time (default 3600 seconds) expires.

Edit: I'm actually surprised myself that download url is cached, versus just images using url (public or signed).  At a quick glance it appears to only be at the CDN level as a different browser also fetches the deleted file.  Cache busting with ?bust=123 at end of path works and file is not found.

**Reply by EdgarsLv** [reply]
I did clear cache with hard reload few times.

**Reply by GaryAustin1** [reply]
I have generated an issue in storage for at least clarification as seems strange for download API function to be cached.
I also verified after an hour the function is no longer cached in the CDN.
https://github.com/supabase/storage-api/issues/116
